modules/nf-core/manta/tumoronly/tests/main.nf.test.snap

```diff
@@ -112,6 +112,14 @@
         },
         "timestamp": "2024-03-21T14:44:15.752887"
     },
+    "bed_version": {
+        "content": null,
```

Comment:
Seems like something is wrong with the versions file

Reply:
yeah, it should be versions, no?

Reply:
Fixed with new assertions